### DISCUSSION TITLE
Direction of Pimcore: mail/dev mode settings, installer, MySQL 8

### DISCUSSION THREAD

**Discussion by Myrmidonut** [pimcore/pimcore#10541] [General]
Hey there!

I got a few questions regarding these things that Pimcore X introduced:

Why remove the mail settings and the dev mode out of the pimcore admin ui? Toggling dev mode on and off has turned into a hassle now. Mail settings is fine I guess because you dont change them all the time. But dev mode? What is the reasoning for doing that? It is very clunky to turn dev mode on and off at the moment.

Since Pimcore is now requiring MySQL 8 it is not compatible with MAMP anymore. Which means local DBs are a thing of the past now unless you run your own custom local webserver. Are you actually needing MySQL 8 features? A hint by the installer would be nice, it just mentions something about RECURSIVE REQUIRED, had to google around a bit until I realized that it is MySQL 8 related.

Speaking of the installer: why is the option removed to set the host and port for the database. The installer in Pimcore 6 was able to ask me that, if I remember correctly. As you can guess, this is closely related to the MySQL 8 issue, because we are now running our local dev environments with a remote DB. I know the installer can work through that database host/port thing if you set the correct parameters when launching it.

Are there any plans on giving us these features back? It feels like a downgrade from a setup point of view. Maybe you can give us an insight on why these changes happened.

**Comment by dpfaffenbauer** [comment]
things evolve, and so does Pimcore. you then could also ask why PHP 8 is the min requirement. you have to take these steps from time to time. I don't see any problem with that.

**Reply by Myrmidonut** [reply]
I agree that MySQL 8 and PHP 8 are just steps that will happen over the time of a product. Thats fine.

My main complaints are the system settings and the installer.
